.storybook/theme.js

```diff
@@ -5,5 +5,5 @@ export default create({
   fontBase: '"M PLUS Rounded 1c", sans-serif',
   brandTitle: 'Matte',
   brandImage: 'images/matte-logo-storybook.png',
-  brandUrl: '?path=/docs/*',
-})
+  brandUrl: '?path=/story/design-system-intro--page',
```

Comment:
Is the double - a typo? -> `intro--page`

Reply:
It's a storybook thing

Reply:
great then